modules-js/next-client-common/src/ScreenReaderSupport.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
This is a longer conversation for another time - and I need to think more on this, and how our apps’ UIs have been assembled - but would this component be providing anything different or additional from what we’d provide a sighted user in the form of an alert or notification of some kind? I’m inclined to prefer a single element that provides the same information to any user, with the ability to add addt’l affordances for screen readers if appropriate.

Reply:
Is this sorted out to your satisfaction?